core/src/test/java/com/graphhopper/routing/EdgeBasedRoutingAlgorithmTest.java

```diff
@@ -143,11 +142,15 @@ private void initTurnRestrictions(Graph g) {
     }
 
     private Weighting createWeighting() {
-        return createWeighting(carEncoder, 40);
+        return createWeighting(carEncoder, Double.POSITIVE_INFINITY);
     }
 
     private Weighting createWeighting(FlagEncoder encoder, double uTurnCosts) {
-        return new TurnWeighting(new FastestWeighting(encoder), tcs).setDefaultUTurnCost(uTurnCosts);
+        TurnWeighting result = new TurnWeighting(new FastestWeighting(encoder), tcs);
+        if (!Double.isInfinite(uTurnCosts)) {
+            result.setUTurnCost(uTurnCosts);
+        }
```

Comment:
Can this be moved into the new constructor?

Reply:
yes this was unnecessary anyway.